Frontend/ui-common/src/Components/ChooseVehicle/View.purs

```diff
@@ -70,15 +71,19 @@ vehicleDetailsView push config =
     [ textView
         $ [ width WRAP_CONTENT
           , height WRAP_CONTENT
-          , text
-              $ case config.vehicleVariant of
-                  "AUTO_RICKSHAW" -> "Auto Rickshaw"
-                  "TAXI" -> "Non-AC Taxi"
-                  "TAXI_PLUS" -> "AC Taxi"
-                  "SEDAN" -> "Sedan"
-                  "SUV" -> "SUV"
-                  "HATCHBACK" -> "Hatchback"
-                  _ -> "Non-AC Taxi"
+          , text $ case (getMerchant FunctionCall) of
+                      YATRISATHI -> case config.vehicleVariant of
```

Comment:
don't add merchant check. get the value from config.

Reply:
soon